<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Budget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
    <item>
     <widget class="QTreeView" name="accountsTree"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="barsLayout"/>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>988</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionCloseFile"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuManage">
    <property name="title">
     <string>Manage</string>
    </property>
    <addaction name="actionManageAccounts"/>
    <addaction name="actionManageCategories"/>
    <addaction name="actionBudget"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuManage"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFile"/>
   <addaction name="separator"/>
   <addaction name="actionManageAccounts"/>
   <addaction name="actionManageCategories"/>
   <addaction name="actionBudget"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionManageAccounts">
   <property name="icon">
    <iconset resource="iconset.qrc">
     <normaloff>:/icons/dollar178.svg</normaloff>:/icons/dollar178.svg</iconset>
   </property>
   <property name="text">
    <string>Accounts</string>
   </property>
   <property name="toolTip">
    <string>Accounts</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="iconset.qrc">
     <normaloff>:/icons/door24.svg</normaloff>:/icons/door24.svg</iconset>
   </property>
   <property name="text">
    <string>Open a file</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="icon">
    <iconset>
     <normaloff>:/iconset/iconset/svg/add189.svg</normaloff>:/iconset/iconset/svg/add189.svg</iconset>
   </property>
   <property name="text">
    <string>New file</string>
   </property>
  </action>
  <action name="actionCloseFile">
   <property name="text">
    <string>Close file</string>
   </property>
   <property name="toolTip">
    <string>Close file</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionManageCategories">
   <property name="icon">
    <iconset resource="iconset.qrc">
     <normaloff>:/icons/bookmark49.svg</normaloff>:/icons/bookmark49.svg</iconset>
   </property>
   <property name="text">
    <string>Categories</string>
   </property>
   <property name="toolTip">
    <string>Categories</string>
   </property>
  </action>
  <action name="actionBudget">
   <property name="icon">
    <iconset resource="iconset.qrc">
     <normaloff>:/icons/left225.svg</normaloff>:/icons/left225.svg</iconset>
   </property>
   <property name="text">
    <string>Budget</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="iconset.qrc"/>
 </resources>
 <connections/>
</ui>
